src/decomp.jl

```diff
@@ -142,13 +142,21 @@ function _factorize_from_right_svd(A::ITensor,
   return FU,FV,spec,commonindex(FU,FV)
 end
 
+# Make the perturbation to the density matrix used in "noise term" DMRG
+# This assumes that A comes in with no primes
+# If it doesn't, I expect A² += drho later to fail
+function deltaRho(A :: ITensor, nt :: ITensor, is)
```

Comment:
Could we rename this to `deltarho`? We are trying to follow the Julia convention of all lower case names.

Reply:
I wondered what the story was with `lproj` etc. Should be fixed in the latest.